docs/configuration/project-config.rst

```diff
@@ -40,7 +37,7 @@ Project Config Example
     config = ProjectConfig(
         dbt_project_path="/path/to/dbt/project",
         models_relative_path="models",
-        seeds_relative_path="data",
+        seeds_relative_path="seeds",
```

Comment:
This path was intentionally different than the default seeds, we shouldn't revert to the default, we could add something like `custom_seeds_folder` if you think it would be more explicit.

Reply:
The value corresponds to an incorrect default used for `seeds_relative_path`, so I assumed it is also incorrect. If the difference is intentional, should I also change `models_relative_path` and `snapshots_relative_path` to non-default values here or remove them from the example?

Reply:
The custom values have been changed as suggested in the comment.